docs/use-cases.rst

```diff
@@ -1,501 +0,0 @@
-a given bytecode, whether or not the |LinkReferences| have values or match.
-
-Query known instances on a given network
-````````````````````````````````````````
-
-Users should be able to see, at a glance, all |ContractInstances| on a given
-|Network|. This can be useful for validating migration state, or for easy
-listing of address / ABI information, to present/share externally.
-
-Query all networks for a given type
-```````````````````````````````````
-
-Developers often write applications so that each |ContractType| has a singleton
-deployed |ContractInstance|. Tooling should enable this first-class nature of
-contract types, and users should be able to track the various instances across
-all known |Networks|.
-
-This is particularly useful when creating front-end applications, referencing
-instances across networks with a single type description.
-
-Query all networks for instances with a given role
-``````````````````````````````````````````````````
-
-In cases where a particular |ContractType| is not deployed as a singleton,
-tooling should provide a mechanism by which users can deploy multiple
-|ContractInstances|, identified in some way distinct from type (i.e., *role*)
-
-As a result, users should be able to use this distinct identifier to query
-for analogous instances across |Networks|, instead of relying solely on the
-type.
-
-
-Interacting with deployed instances
------------------------------------
-
-Besides metadata information, users must be able to retrieve runtime state
-information about |ContractInstances| and to be able to write to those
-instances over the network.
-
-
-.. uml::
-
-  left to right direction
-
-  :User: as :User:
-
-  rectangle Metadata << external >> {
-    (Read ABI) as (ReadABI)
-    (Read instance) as (ReadInstance)
-    (ReadInstance) -> ReadABI
-  }
-
-  rectangle Interaction {
-    (Call read-only method) as (Call)
-    (Invoke method via a transaction) as (SendTransaction)
-    (Send ETH to a contract instance) as (Send)
-
-    Call --> ReadInstance
-    SendTransaction --> ReadInstance
-  }
-
-  User -- Call
-  User -- SendTransaction
-  User -- Send
-
-Call read-only method
-`````````````````````
-
-Almost every smart contract provides mechanisms for viewing information
-about a |ContractInstance|'s current state, including public storage variables
-and computed data views. These interfaces are specified for each instance's
-|ContractType|, and described in its |ABI|.
-
-Users must be able to call these read-only methods and obtain their results.
-
-Invoke method via a transaction
-```````````````````````````````
-
-Similarly to calls, most contracts provide interface methods for modifying
-|ContractInstance| state. Users must be able to execute the allowable
-"write" operations for a contract instance.
-
-Send ETH to a contract instance
-```````````````````````````````
-
-Many smart contracts expose methods for receiving payment in the form or ether
```

Comment:
This should be "in the form *of* ether"

Reply:
Well, that file was deleted, so I think the typo no longer exists :)